go/teams/handler.go

```diff
@@ -400,29 +400,36 @@ func handleSeitanSingle(ctx context.Context, g *libkb.GlobalContext, team *Team,
 	if category != keybase1.TeamInviteCategory_SEITAN {
 		return fmt.Errorf("HandleTeamSeitan wanted to claim an invite with category %v", category)
 	}
+	err = handleSeitanSingleV1(ctx, g, team, invite, seitan)
+	if err != nil {
+		err = handleSeitanSingleV2(ctx, g, team, invite, seitan)
```

Comment:
And if we used same struct for both we can decode once and decide based on version field

Reply:
yes much cleaner!